commands/create.js

```diff
@@ -258,7 +259,9 @@ function api(opts = {}) {
 			.then(() => {
 				if (opts.local) {
 					console.log('Installing @enact/cli locally. This might take a couple minutes.');
-					return npmInstall(opts.directory, opts.verbose, '--save-dev', ENACT_DEV_NPM);
+					return npmInstall(opts.directory, opts.verbose, '--save', CORE_JS_NPM).then(() =>
```

Comment:
Any reason to not just rename `ENACT_DEV_NPM` to `ENACT_LOCAL_DEPS` and just combine them together into one variable and call?

Reply:
`CORE_JS_NPM` would be a dependency and ENACT_DEV_NPM would be a devDependency.  Don't believe you can use both `--save` and `--save-dev` together in 1 command without one overriding the other.

Reply:
oh.  :nothingtodohere:

Reply:
As it stands, they're landing in different spots (dependencies vs devDependencies) so I don't think that'd be compatible.